tutorials/distributed_data_classification/multiple_quality_models.ipynb

```diff
@@ -0,0 +1,697 @@
+    "        batch_size=1024,\n",
+    "        pred_column=pred_column,\n",
+    "        prob_column=prob_column,\n",
+    "    )\n",
+    "    dataset = classifier(dataset=dataset)\n",
+    "    fold += 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "GPU: 0, Part: 0: 100%|██████████| 10/10 [00:04<00:00,  2.44it/s]\n",
+      "GPU: 0, Part: 0: 100%|██████████| 10/10 [00:00<00:00, 30.69it/s]\n",
+      "GPU: 0, Part: 0: 100%|██████████| 10/10 [00:00<00:00, 29.82it/s]\n",
+      "GPU: 0, Part: 0: 100%|██████████| 10/10 [00:00<00:00, 31.67it/s]\n",
+      "GPU: 0, Part: 0: 100%|██████████| 10/10 [00:00<00:00, 82.84it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 613 ms, sys: 361 ms, total: 974 ms\n",
+      "Wall time: 10.7 s\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "GPU: 0, Part: 0: 100%|██████████| 10/10 [00:00<00:00, 31.16it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>text</th>\n",
+       "      <th>quality_prob_0</th>\n",
+       "      <th>quality_pred_0</th>\n",
+       "      <th>quality_prob_1</th>\n",
+       "      <th>quality_pred_1</th>\n",
+       "      <th>quality_prob_2</th>\n",
+       "      <th>quality_pred_2</th>\n",
+       "      <th>quality_prob_3</th>\n",
+       "      <th>quality_pred_3</th>\n",
+       "      <th>quality_prob_4</th>\n",
+       "      <th>quality_pred_4</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Quantum computing is set to revolutionize the ...</td>\n",
+       "      <td>[0.3728572130203247, 0.499016135931015, 0.1281...</td>\n",
+       "      <td>Medium</td>\n",
+       "      <td>[0.3728572130203247, 0.499016135931015, 0.1281...</td>\n",
+       "      <td>Medium</td>\n",
+       "      <td>[0.3728572130203247, 0.499016135931015, 0.1281...</td>\n",
+       "      <td>Medium</td>\n",
+       "      <td>[0.3728572130203247, 0.499016135931015, 0.1281...</td>\n",
+       "      <td>Medium</td>\n",
+       "      <td>[0.3728572130203247, 0.499016135931015, 0.1281...</td>\n",
+       "      <td>Medium</td>\n",
```

Comment:
It looks like all 5 models produce the exact same probabilities, is this expected?